changelogs/current.yaml

```diff
@@ -116,6 +116,11 @@ minor_behavior_changes:
 - area: logging
   change: |
     changed category name for access log filter extensions to ``envoy.access_loggers.extension_filters``.
+- area: http
+  change: |
+    header invert match rule will apply when the header is not present and the header match rule is not present type.
+    This behavior change can be reverted by setting runtim guard
```

Comment:
runtim -> runtime

also I'd be inclined to put this in the bugfix section

Reply:
Thank you! Fixed the changelog.